projects/openapi-cli/src/commands/capture.ts

```diff
@@ -29,27 +30,46 @@ export async function captureCommand(): Promise<Command> {
   let certCommand = await captureCertCommand();
```

Comment:
This looks like it's registered twice (here, and in the `optic init.ts` file)

Reply:
good catch. removed and tested. I'm not sure why it was done that way before